aws/datadog_integration_role.yaml

```diff
@@ -147,6 +81,7 @@ Resources:
                 'sts:ExternalId': !Ref ExternalId
       Path: /
       RoleName: !Ref IAMRoleName
+      ManagedPolicyArns: !If [ShouldInstallResourcePolicy, ['arn:aws:iam::aws:policy/SecurityAudit'], !Ref AWS::NoValue]
```

Comment:
Because of the param name change, let's also change `ShouldInstallResourcePolicy` to `ShouldInstallCSPMPolicy`

Reply:
CSPM will have a lot of overlapping permissions with other resources products. Do we want to have each product require a separate list of permissions? If we keep it as `ResourceCollection` , we could say provide resource collection permissions to use any of ... products